src/robot/libraries/Reserved.py

```diff
@@ -23,4 +23,8 @@ def get_keyword_names(self):
         return RESERVED_KEYWORDS
 
     def run_keyword(self, name, args):
-        raise Exception("'%s' is a reserved keyword" % name.title())
+        error = "'%s' is a reserved keyword." % name
+        if name in ['else', 'else if']:
+            error += " '%s' must be in uppercase (%s) when used as a marker with 'Run Keyword  If'." % (
```

Comment:
There's actually no need to repeat the keyword name here when the original `error` already contains it. The addition could start with ` It must be in ...`.